### ISSUE TITLE
diff_mq_2_1 failing on ppc64el

### ISSUE THREAD

**Issue by d-torrance** [algebraic-solving/msolve#144]
I just uploaded msolve 0.6.6 to Debian, and one of the tests is failing during the 64-bit PowerPC build:

```diff
FAIL: test/diff/diff_mq_2_1.sh
==============================

Warning: this part of the code has not been tested intensively
Something should be checked
3c3
< 2, 
---
> 1, 
7,8c7,8
< [[2,
< [20, 6, 1]],
---
> [[1,
> [159, 1]],
FAIL test/diff/diff_mq_2_1.sh (exit status: 42)
```

Full log: https://buildd.debian.org/status/fetch.php?pkg=msolve&arch=ppc64el&ver=0.6.6-1&stamp=1719514089&file=log

**Comment by mohabsafey** [comment]
I will have a look at this next week

**Comment by d-torrance** [comment]
A [subsequent ppc64el build](https://buildd.debian.org/status/fetch.php?pkg=msolve&arch=ppc64el&ver=0.6.6-1&stamp=1719754444&raw=0) was successful.  Is the algorithm not deterministic?

**Comment by ederc** [comment]
Issue is fixed via #154.

We are preparing a new minor release.
